Say Their Names/Source/Utils/Extentions/UICollectionView+Extentions.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  UICollectionView+Extentions.swift
+//  Say Their Names
+//
+//  Created by Thomas Murray on 03/06/2020.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+extension UICollectionView {
```

Comment:
Let's keep this code with the currently existing extensions in `UICollectionView++`

Reply:
Moved into the suggested file